diff --git a/ffbuild/common.mak b/ffbuild/common.mak
index 0a60d01623..d3aa1204e1 100644
--- a/ffbuild/common.mak
+++ b/ffbuild/common.mak
@@ -115,9 +115,9 @@ COMPILE_LASX = $(call COMPILE,CC,LASXFLAGS)
 $(BIN2CEXE): ffbuild/bin2c_host.o
 	$(HOSTLD) $(HOSTLDFLAGS) $(HOSTLD_O) $^ $(HOSTEXTRALIBS)
 
-RUN_BIN2C = $(BIN2C) $(patsubst $(SRC_PATH)/%,$(SRC_LINK)/%,$<) $@ $(subst .,_,$(basename $(notdir $@)))
-RUN_GZIP  = $(M)gzip -nc9 $(patsubst $(SRC_PATH)/%,$(SRC_LINK)/%,$<) >$@
-RUN_MINIFY = $(M)sed 's!/\\*.*\\*/!!g' $< | tr '\n' ' ' | tr -s ' ' | sed 's/^ //; s/ $$//' > $@
+RUN_BIN2C = $(BIN2C) $(subst \,/,$(patsubst $(SRC_PATH)/%,$(SRC_LINK)/%,$<)) $(subst \,/,$@) $(subst .,_,$(basename $(notdir $@)))
+RUN_GZIP = $(M)gzip -nc9 $(subst \,/,$(patsubst $(SRC_PATH)/%,$(SRC_LINK)/%,$<)) > $(subst \,/,$@)
+RUN_MINIFY = $(M)sed 's!/\\*.*\\*/!!g' $(subst \,/,$<) | tr '\n' ' ' | tr -s ' ' | sed 's/^ //; s/ $$//' > $(subst \,/,$@)
 %.gz: TAG = GZIP
 %.min: TAG = MINIFY
 
